config_db_migrate/versions/67ee7b73a1dc_new_table_for_notification_banner_string.py

```diff
@@ -0,0 +1,37 @@
+"""New Table for notification banner string
+
+Revision ID: 67ee7b73a1dc
+Revises: 3335ff7593cc
+Create Date: 2019-01-30 11:14:46.327676
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '67ee7b73a1dc'
+down_revision = '3335ff7593cc'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    config_table = op.create_table('configuration',
+    sa.Column('lock', sa.Boolean(), server_default=sa.sql.false(), nullable=False),
```

Comment:
Why do we need an extra lock column?

Reply:
The lock is the guard (primarykey and a check constraint) for making sure that the config table has only one row.

Reply:
If there will be several types of notification as suggested in the API, it will be necessary to insert multiple rows. I assume this can be solved later by schema migration.